cmd/queue/main.go

```diff
@@ -157,17 +155,15 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 
 		// Enforce queuing and concurrency limits.
 		if breaker != nil {
-			cf := noop
+			var waitSpan *trace.Span
 			if tracingEnabled {
-				ctx, waitSpan := trace.StartSpan(r.Context(), "queueWait")
-				r = r.WithContext(ctx)
```

Comment:
Doesn't dropping this mean we won't root child spans?  That might be the right thing anyways since the calls are sequenced... 🤔

Reply:
Read PR description 👍